test/functional/CommandCallFunctional.js

```diff
@@ -68,15 +69,15 @@ describe('CommandCall Functional Tests', () => {
         connection.add(new CommandCall({ command: 'system -i wrksyssts', type: 'sh' }));
         connection.run((error, xmlOut) => {
           expect(error).to.equal(null);
-          const results = xmlToJson(xmlOut);
           // xs does not return success property for iSh or iQsh
           // but on error data property = '\n'
           // so lets base success on contents of data.
-          results.forEach((result) => {
-            expect(result.data).not.to.equal('\n');
-            expect(result.data).to.match(/(System\sStatus\sInformation)/);
+          parseString(xmlOut, (parseError, result) => {
+            expect(parseError).to.equal(null);
+            expect(result.myscript.sh[0]._).not.to.equal('\n');
+            expect(result.myscript.sh[0]._).to.match(/(System\sStatus\sInformation)/);
```

Comment:
Given the second check, seems the first check is pointless. Also, the comment above needs to be removed or adjusted.